### PR TITLE
Related #5517: Selection inconcistency with translation menu

### PR DESCRIPTION
**Pull request by kumy**
Fix what is described as "Behavior 1" in the issue.

### CODE REVIEW

**Comment by kumy** [comment]
Anyone willing to check?

File: main/src/cgeo/geocaching/CacheDetailActivity.java [hunk 1]
```diff
@@ -1949,7 +1950,8 @@ public boolean onMenuItemClick(MenuItem menuItem) {
 
     @Override
     public void onSupportActionModeStarted(ActionMode mode) {
-        if (mActionMode == null && selectedTextView != null) {
+        if (mSelectionModeActive == true && selectedTextView != null) {
```
Comment (samueltardieu) [inline main/src/cgeo/geocaching/CacheDetailActivity.java]: Please don't compare to `true`. `if (mSelectionModeActive && selectedTextView != null)` is enough.

File: main/src/cgeo/geocaching/CacheDetailActivity.java [hunk 2]
```diff
@@ -1968,7 +1970,9 @@ public void onSupportActionModeFinished(ActionMode mode) {
         mActionMode = null;
         if (selectedTextView != null)
             selectedTextView.setWindowFocusWait(false);
-        selectedTextView = null;
+        if (mSelectionModeActive == false) {
```
Comment (samueltardieu) [inline main/src/cgeo/geocaching/CacheDetailActivity.java]: `if (!mselectionModeActive)`

File: main/src/cgeo/geocaching/CacheDetailActivity.java [hunk 3]
```diff
@@ -1934,6 +1934,7 @@ public static void startActivity(final Context context, final String geocode, fi
     }
 
     private ActionMode mActionMode = null;
+    private boolean mSelectionModeActive = false;
```
Comment (samueltardieu) [inline main/src/cgeo/geocaching/CacheDetailActivity.java]: We don't usually follow the convention `mXXX` for member fields. I'd prefer that we stay consistent and use `selectionModeActive` instead.

**Comment by kumy** [comment]
Thanks ;)

**Pull request closed without merging**